omniledger/service/service.go

```diff
@@ -799,6 +800,7 @@ clientTransactions:
 			scs, cout, err := s.executeInstruction(cdbI, cin, instr)
 			if err != nil {
 				log.Errorf("%s: Call to contract returned error: %s", s.ServerIdentity(), err)
+				ctsBad = append(ctsBad, ct)
```

Comment:
Shouldn't we also append `ctsBad` in the `scs` loop too? Again, it might be better to just return the remaining cts, probably look at the latest index and do something like `ctsRem := cts[latestIdx+1:]` after the outer for loop.